pkg/container/types/date.go

```diff
@@ -43,6 +43,33 @@ const (
 	Saturday
 )
 
+// String returns the English name of the day ("Sunday", "Monday", ...).
+func (d Weekday) String() string {
+	if Sunday <= d && d <= Saturday {
+		return longDayNames[d]
+	}
+	buf := make([]byte, 20)
+	n := fmtInt(buf, uint64(d))
+	return "%!Weekday(" + string(buf[n:]) + ")"
```

Comment:
Wait -- what is this?

Reply:
This code is used to handle the excessive weekday. I will optimize it